lcls_naming_tool/ccc_taxon.json

```diff
@@ -0,0 +1,188 @@
+	"FTC": "Flow meter 4-20ma Analog",
+	"GBC": "Gauge Beam Containment (System)",
+	"GCC": "Gauge Cold Cathode",
+	"GCI": "Gauge Controller, Ionivac ",
+	"GCM": "Gauge Capacitance Manometer",
+	"GCP": "Gauge Cold(hot) Cathode/Pirani Combination",
+	"GCT": "Gauge Controller",
+	"GFM": "Gas Flow Meter",
+	"GFS": "Gauge fast shutter",
+	"GHC": "Gauge Hot Cathode",
+	"GHP": "Gauge Hot Cathode/Pirani Combination",
+	"GIGE": "GigE Cameras",
+	"GMP": "Gauge Capacitance Manometer",
+	"GPI": "Gauge Pirani",
+	"GPS": "Gauge pressure switch",
+	"GPT": "Gauge  Pressure Trancducer",
+	"GSR": "Gauge Spinning Rotor",
+	"GTC": "Gauge Thermocouple",
+	"HPS": "High performance system",
+	"HMD": "",
+	"HTR": "Heating element",
+	"ICT": "ICT circuit breaker module",
+	"INM": "INclinoMeter/tiltmeter",
+	"IOC": "Controller, SLAC supplied for Inputs and Outputs",
+	"LCN": "Laser CoNtroller",
+	"LCT": "Laser Curtain Sensor (previously called CTN)",
+	"LDM": "XTDS ladder mover",
+	"LDT": "Leak DeTector",
+	"LIC": "Laser Incoupling Mirrors",
+	"LIM": "Limit switch(es)",
+	"LOC": "Laser Optical Chopper",
+	"LOT": "Laser Optical Table",
+	"LPS": "Laser power supply interlock",
+	"LSC": "Laser Shutter Controller",
+	"LST": "Laser Shutter",
+	"LSW": "Liquid Level Switch",
+	"LTP": "Laser Touch Panel",
+	"LVDT": "LVDT",
+	"LWP": "Laser waveplate attenuator",
+	"MCJ": "Motion Controller PiezoSystemJENA",
+	"MCL": "Motion Controller Labview",
+	"MCM": "Motion Controller Micronix",
+	"MCP": "Motion Controller Pico",
+	"MCQ": "Motion Controller sQuareone",
+	"MCT": "Motion Controller smaracT",
+	"MCZ": "Motion Controller PiezoSystem pi",
+	"MFW": "Motion Filter Wheel",
+	"MMD": "Motion Motor DC",
+	"MMS": "Motion Motor Stepper/Servo",
+	"MMT": "Motion Motor smaracT",
+	"MMZ": "Motion Motor pieZo",
+	"MPA": "Motion Pneumatic Actuator",
+	"MRD": "Motion Rail Driver",
+	"MRP": "Motion Rail PLC",
+	"OSC": "remote controlled OSCilliscope",
+	"PCI": "Pump Controller Ion",
+	"PCL": "Personal Computer Laptop",
+	"PCS": "Pump controller Sublimation",
+	"PCT": "Pump Controller Turbo",
+	"PGC": "Pulser generator chassis",
+	"PGP": "PGP Card (KCU1500 or other)",
+	"PGT": "Getter Pump",
+	"PHV": "Power supply high voltage",
+	"PID": "Generic Closed Loop PID Controller (Pressure, flow, etc.)",
+	"PIN": "Pump Ion Negative",
+	"PIP": "Positive Ion Pump",
+	"PLC": "Programmable logic controller",
+	"PLV": "Power supply Low Voltage",
+	"PMF": "Mechanical foreline pump",
+	"PMPS": "Photon Machine Protection System (component)",
+	"PNL": "Generic panel",
+	"POT": "Potentiometer",
+	"PPC": "Pump - Peristaltic",
+	"PPL": "Patch Panel",
+	"PPM": "Power and Profile Monitor",
+	"PRO": "Vacuum Roughing Pump",
+	"PROF": "Profile Monitor",
+	"PRSM": "Pressure Meter",
+	"PRT": "Pump Roots",
+	"PSH": "Phase SHifter",
+	"PSI": "Power Supply hIgh current",
+	"PSM": "Power Supply Motor",
+	"PSP": "Power Supply PMC-100",
+	"PST": "Power Supply Pump",
+	"PSV": "Power Supply high Voltage",
```

Comment:
There are a few weird marks on the source page, such as crossing out this entry, highlighting other entries in odd colors, etc., so we should double-check with the team if any of these CCCs should be removed.

Reply:
Will follow up on this in the next PR.